dashboard/app/models/vocabulary.rb

```diff
@@ -125,6 +125,17 @@ def serialize_scripts
     end
   end
 
+  def copy_to_course_version(destination_course_version)
+    return self if course_version == destination_course_version
+    persisted_vocab = Vocabulary.where(word: word, course_version_id: destination_course_version.id).first
+    if persisted_vocab && !!persisted_vocab.common_sense_media == !!common_sense_media
+      persisted_vocab
+    else
+      copied_vocab = Vocabulary.create!(word: word, definition: definition, common_sense_media: common_sense_media, course_version_id: destination_course_version.id)
+      copied_vocab
+    end
+  end
```

Comment:
thank you for extracting this. I looked for other stuff to extract out of copy_to_unit but I didn't see anything that could be pulled out super cleanly.

Reply:
I had already extracted this same logic for resources so I'm not sure why I didn't do the same for vocab. 🤷‍♀️  Thanks for looking for more things to extract -- always happy to pull more things out if possible :)